code/src/components/TaskList.js

```diff
@@ -0,0 +1,108 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import tasks from 'reducers/tasks'
+import styled from 'styled-components'
+import { Button } from 'components/GlobalStyles'
+import { FaTrash } from 'react-icons/fa';
+import moment from 'moment';
+
+const TaskList = () => {
+  const taskList = useSelector((store) => store.tasks.items);
+  const dispatch = useDispatch();
+
+  const onDeleteItemClick = (todoIndex) => {
+    dispatch(tasks.actions.deleteItem(todoIndex));
+  };
+
+  const onCompleteToggle = (id) => {
+    dispatch(tasks.actions.toggleItem(id))
+  };
+
+  return (
+    <StyledList>
+      {taskList.map((singleTask, index) => {
+        return (
+          <ListWrapper>
+            <div>
+              <input
+                className="checkbox"
+                name="tasks"
+                type="checkbox"
+                checked={singleTask.complete}
+                onChange={() => onCompleteToggle(singleTask.id)} />
+              <label htmlFor={singleTask.id}>{singleTask.task}</label>
+            </div>
+            <TasksText>{singleTask.text}</TasksText>
+            <DateAndDeleteBtn>
+              <Date>{moment(tasks.date).format('ddd, MMM Do YYYY @ hh:mm a')}</Date>
+              <DeleteBtn type="button" onClick={() => onDeleteItemClick(index)}> <FaTrash /></DeleteBtn>
+            </DateAndDeleteBtn>
+          </ListWrapper>
+        )
+      })}
+    </StyledList>
+  )
+}
+
+export default TaskList
+
+export const StyledList = styled.section`
```

Comment:
Hmm, if you add `background-color: #fbdfcf;` to this section it might remove the issue with clipping the color of the background when you have more than 3 tasks.